go/host/rpc/clientapi/client_api_eth.go

```diff
@@ -183,27 +182,29 @@ func (api *EthereumAPI) FeeHistory(context.Context, rpc.DecimalOrHex, rpc.BlockN
 func headerWithHashesToMap(headerWithHashes *common.HeaderWithTxHashes) map[string]interface{} {
 	header := headerWithHashes.Header
 	return map[string]interface{}{
```

Comment:
Reorders these fields to match their order in the `common.Header` struct, and adds a few missing fields.